metricbeat/docker-compose.yml

```diff
@@ -17,11 +17,11 @@ services:
 
   # Used by base tests
   elasticsearch:
-    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-8.1.2}-1
+    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-8.2.1}-1
```

Comment:
This is a wrong version, it's supposed to be 8.4.0, I think the automation didn't work right.

Reply:
Automation worked as expected, those files point to the `latest released version`. There are likely some specific reason in the `ingest-dev` repo (which it's a private repo)

https://github.com/elastic/beats/pull/31059 was also another conversation when `8.3.0` was created.

The script does the follow:

```
CURRENT_RELEASE=*** - version we want to prepare.
MAJOR ?= $(shell echo $(CURRENT_RELEASE) | cut -f1 -d.)
MINOR ?= $(shell echo $(CURRENT_RELEASE) | cut -f2 -d.)
PATCHNUM ?= $(shell echo $(CURRENT_RELEASE) | cut -f3 -d.)
LATEST_RELEASE ?= $(MAJOR).$(MINOR).$(shell expr $(PATCHNUM) - 1)

...
$(SED) -E -e "s#:-[0-9]+\.[0-9]+\.[0-9]+}#:-$(LATEST_RELEASE)}#g" metricbeat/docker-compose.yml;

```


The automation happened in [here](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2FRelease%2Fbeats-release-minor-major/detail/beats-release-minor-major/19/pipeline/174#step-187-log-16)

```
[2022-05-25T17:47:09.024Z] RELEASE_TYPE=minor
[2022-05-25T17:47:09.024Z] BASE_BRANCH=main
[2022-05-25T17:47:09.024Z] RELEASE_BRANCH=8.3
[2022-05-25T17:47:09.024Z] LATEST_RELEASE=8.2.1
[2022-05-25T17:47:09.024Z] NEXT_MINOR=8.4.0
[2022-05-25T17:47:09.024Z] NEXT_RELEASE=8.3.1
```

and then the changes happened afterwards:

```
[2022-05-25T17:54:43.118Z] -        KIBANA_VERSION: ${KIBANA_VERSION:-8.1.2}
[2022-05-25T17:54:43.118Z] +        KIBANA_VERSION: ${KIBANA_VERSION:-8.2.1}
[2022-05-25T17:54:43.118Z]      depends_on:
[2022-05-25T17:54:43.118Z]        - elasticsearch
[2022-05-25T17:54:43.118Z]      ports:
[2022-05-25T17:54:43.384Z] [update_dev_env_8.4.0 ee36454b91] [Release] update test environment
[2022-05-25T17:54:43.384Z]  3 files changed, 11 insertions(+), 11 deletions(-)
[2022-05-25T17:54:43.384Z] make[3]: Entering directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:43.384Z] ************************************************************
[2022-05-25T17:54:43.384Z] ********************** Git Changes - END *************************
[2022-05-25T17:54:43.384Z] ************************************************************
[2022-05-25T17:54:43.384Z] 
[2022-05-25T17:54:43.384Z] make[3]: Leaving directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:43.384Z] make[2]: Leaving directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:43.384Z] make[2]: Entering directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:43.384Z] ************************************************************
[2022-05-25T17:54:43.384Z] ********************** Update test environments - END *************************
[2022-05-25T17:54:43.384Z] ************************************************************
[2022-05-25T17:54:43.384Z] 
[2022-05-25T17:54:43.384Z] make[2]: Leaving directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:43.384Z] make[1]: Leaving directory '/var/lib/jenkins/workspace/Beats/Release/beats-release-minor-major/src/github.com/elastic/ingest-dev/release_scripts'
[2022-05-25T17:54:44.774Z] remote: 
[2022-05-25T17:54:44.774Z] remote: Create a pull request for 'update_dev_env_8.4.0' on GitHub by visiting:        
[2022-05-25T17:54:44.775Z] remote:      https://github.com/elastic/beats/pull/new/update_dev_env_8.4.0        
[2022-05-25T17:54:44.775Z] remote: 
[2022-05-25T17:54:44.775Z] remote: GitHub found 3 vulnerabilities on elastic/beats's default branch (2 high, 1 low). To find out more, visit:        
[2022-05-25T17:54:44.775Z] remote:      https://github.com/elastic/beats/security/dependabot        
[2022-05-25T17:54:44.775Z] remote: 
[2022-05-25T17:54:44.775Z] To github.com:elastic/beats.git
[2022-05-25T17:54:44.775Z]  * [new branch]            HEAD -> update_dev_env_8.4.0
[2022-05-25T17:54:44.775Z] Branch 'update_dev_env_8.4.0' set up to track remote branch 'update_dev_env_8.4.0' from 'origin'.
[2022-05-25T17:54:49.012Z] https://github.com/elastic/beats/pull/31758
```

Unfortunately, I don't have enough context about the main reason for using that version... but the output is expected so no issues in the automation at all